packages/react-components/src/components/gantt-chart-output-component.tsx

```diff
@@ -0,0 +1,1712 @@
+import * as React from 'react';
```

Comment:
this component mostly the same as the timegraph-output-component. Could we abstract it to a common base class and have gantt and regular timegraph extend it?